### ISSUE TITLE
installing from cheese shop is broken.

### ISSUE THREAD

**Issue by sivang** [machinalis/quepy#28]
I have installed as so:
pip install quepy

Then:

sivan@metal:~/icrav/code$ quepy
Traceback (most recent call last):
  File "/home/sivan/icrav-venv/bin/quepy", line 27, in <module>
    from docopt import docopt
ImportError: No module named docopt

It appears that this fixes it:
pip install docopt

so docopt needs to be added to the requirements.txt

**Comment by j0hn** [comment]
You're right and that's already fixed on the development version, we haven't released a new version in a while and maybe it's time to do it. 

You might want to install quepy by doing thise:

`pip install git+https://github.com/machinalis/quepy.git`

and you'll get the latest development version.

@rafacarrascosa what do you think? there are a bunch of fixes that are piling up and should be released already.

**Comment by sivang** [comment]
Thanks for the quick reply! I will do so, I love your effort and the library. I can probably help with release and bug fixing like this as a starting point to get involved.

BTW: Can we have a mongodb or any other KeyValue store support? I'm willing to do the coding.

**Comment by sivang** [comment]
also, parsing.py is missing when I do quepy startapp dbpedia, as in the readthedocs tutorial.
